### DISCUSSION TITLE
A new plugin for a oddball task

### DISCUSSION THREAD

**Discussion by Aakashagr** [jspsych/jsPsych#2420] [General]
Hello,

I have created a new plugin (attached as txt file) for an oddball task using a combination of [VSL-grid-plugin](https://github.com/jspsych/jspsych-contrib/blob/main/packages/plugin-vsl-grid-scene/src/index.ts) and plugin-image-button-response.
A few trials (n = 78) of the task can be found [here](https://px716fdpdk.cognition.run/). 

Thanks,
Aakash
[searcharrayplugin.txt](https://github.com/jspsych/jsPsych/files/7793363/searcharrayplugin.txt)

**Comment by jodeleeuw** [comment]
Hi @Aakashagr,

Very cool! Consider submitting it to our [community contributions repository](https://github.com/jspsych/jspsych-contrib).
